### PR TITLE
Fix images in evaluation notebook

### PR DESCRIPTION
**Pull request by Hakimovich99**
a few changes brought by Jan to the notebooks.

### CODE REVIEW

**Comment by RobbeSneyders** [comment]
Hi Hakim, these are the old images being added again, which don't reflect the true pipeline. Let's stick to the ones I added. Also really don't like the gif at the end.

**Comment by Hakimovich99** [comment]
> Hi Hakim, these are the old images being added again, which don't reflect the true pipeline. Let's stick to the ones I added. Also really don't like the gif at the end.

Oh alright alright 

> Hi Hakim, these are the old images being added again, which don't reflect the true pipeline. Let's stick to the ones I added. Also really don't like the gif at the end.

Ah okok, sorry I took them from Jan but didn't know there were other new images. But I actually don't see any image in the notebook from the main branch. Is it only me?

**Comment by RobbeSneyders** [comment]
They seem to be broken indeed, my bad. I posted the excalidraw with the latest images in chat. Can you update them?

**Comment by Hakimovich99** [comment]
> They seem to be broken indeed, my bad. I posted the excalidraw with the latest images in chat. Can you update them?

Sure! Can you pin me pls? Can't find it haha

File: src/evaluation.ipynb [hunk 1]
```diff
@@ -119,14 +111,14 @@
    "outputs": [],
    "source": [
     "!docker compose version\n",
-    "!docker info >> /dev/null && echo \"Docker running\""
```
Comment (RobbeSneyders) [inline src/evaluation.ipynb:122]: Don't know why this was removed, but I don't think we need this output.

File: src/evaluation.ipynb [hunk 2]
```diff
@@ -498,16 +469,26 @@
...
    "source": [
     "Please share your experience or let us know how we can improve through our \n",
     "* [**Discord**](https://discord.gg/HnTdWhydGp) \n",
     "* [**GitHub**](https://github.com/ml6team/fondant)\n",
     "\n",
-    "And of course feel free to give us a [**star** ⭐](https://github.com/ml6team/fondant) if you like what we are doing!"
+    "And of course feel free to give us a [**star** ⭐](https://github.com/ml6team/fondant) if you like what we are doing!\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"../art/feedback_is_important.webp\"/>\n",
```
Comment (RobbeSneyders) [inline src/evaluation.ipynb:482]: Would still remove this :)

**Review by RobbeSneyders — APPROVED** [review]

**Pull request merged**